mix.exs

```diff
@@ -73,10 +73,9 @@ defmodule Membrane.RTP.Plugin.MixProject do
 
   defp deps do
     [
-      {:membrane_core,
-       github: "membraneframework/membrane_core", tag: "v0.9.0-rc.0", override: true},
+      {:membrane_core, "~> 0.9.0", override: true},
```

Comment:
Why override? You should rather update elements. `h264_ffmpeg` has already 0.17 ready for core 0.9